mainnet/16-wysiwyg-invalid-agent-tx/evidence/for.md

```diff
@@ -0,0 +1,9 @@
+Name: @light 
+
+Contact: @light in the Aragon Chat and Aragon Forum
+
+Summary: Transfers control of the `ancash.eth` ENS name from one of our Agent addresses to our other AN Cash Agent address.
+
+Rationale: We decided to use the AN Cash Agent address `0x87f4dc77707cdbedd0069c3cfc1b96f1725f03e0` to hold all of our ENS names.
+
+Description: If approved, this proposal will transfer the controller of the `ancash.eth` ENS name from AN Cash Agent address `0xF46Ac50c67E77Bc5CddDEf0E323963B6F0AEAde8` to the AN Cash Agent address `0x87f4dc77707cdbedd0069c3cfc1b96f1725f03e0`.
```

Comment:
Should we/this Agent be in control of this ENS name?

Reply:
The agent that will transfer the ENS name needs to be the controller of it, yes.